html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+    <style>
+      fieldset {
+        border: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html" method="post"></form>
```

Comment:
The closing tag for the `form` should be at the bottom. All of the content should be inside this `form` element